share/openPMD/json_schema/Makefile

```diff
@@ -0,0 +1,15 @@
+convert := openpmd-convert-json-toml
```

Comment:
Naming it `openpmd-convert-toml-json` seems to be more intuitive for what we use it here, but you mentioned anyway that it works bi-directional?